bin/docker-migrate

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
```

Comment:
Q: Why drop this here? Seems like an unrelated refactoring, bloating an already big migration?

Reply:
point was to remove a never used in isolatoin command because you'd always want to migrate_clickhouse if we're defaulting to that support

Reply:
brought it back for now